tests/pods_test.go

```diff
@@ -0,0 +1,66 @@
+package tests
+
+import (
+	_ "embed"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+//go:embed results/pods_all_namespaces.json
+var expectedGetAllPodsResult string
+
+//go:embed results/pods_velero_namespace.json
+var expectedGetVeleroPodsResult string
+
+//go:embed results/pods_restic_only.json
+var expectedGetResticPodsResult string
+
+var _ = Describe("GET /api/v1/pods", func() {
+	Context("When getting pods in all namespaces", func() {
+		It("Returns all pods", func() {
+			resp, statusCode, err := HTTPExec("GET", fmt.Sprintf("%s/api/v1/pods", apiServerEndpoint), getHeaders)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(statusCode).To(Equal(http.StatusOK))
+			Expect(resp).To(Similar(expectedGetAllPodsResult))
```

Comment:
what's changing here that requires using similar?

Reply:
times are relative, like "pod has been running for 2 hours"